### ISSUE TITLE
Tensorflow 2.3.0 TfLite converter adds ExpandDims

### ISSUE THREAD

**Issue by DominikStachura** [tensorflow/tensorflow#43141]
I have a problem with new version of Tensorflow. I try to deploy my model to embedded system. I have trained Keras model and want to convert it to *.tflite format. Tensorflow 2.2.0 generates .tflite model which is accepted by tflite/micro on embedded system. However Tensorflow 2.3.0 adds some ExpandDims layer at the beggining of the network, and ExpandDims can not e resolved on the tf/lite/micro side. 
Convertion using TF 2.3.0:
![image](https://user-images.githubusercontent.com/58625554/92916599-f3403b00-f42d-11ea-9562-03e6aca59bb0.png)

Convertion using TF 2.2.0:
![image](https://user-images.githubusercontent.com/58625554/92916775-1b2f9e80-f42e-11ea-9ba9-672af0d6281f.png)

We want to move our project from tf 2.2 to tf 2.3 ut for now it is impossible because of this problem.
Here is the code used for convertion:
`        converter = tf.lite.TFLiteConverter.from_keras_model(LOADED_KERAS_MODELl)
        converter.optimizations = [tf.lite.Optimize.DEFAULT]

        quantized_and_pruned_tflite_model = converter.convert()

        quantized_and_pruned_tflite_file = 'model_lite_pruned_microchip.tflite'

        with open(quantized_and_pruned_tflite_file, 'wb') as f:
            f.write(quantized_and_pruned_tflite_model)
        print('Saved quantized and pruned TFLite model to:', quantized_and_pruned_tflite_file)`

Tools versions:
_Python 3.8.5,
Tensorflow 2.2.0/2.3.0
tensorflow-model-optimization 0.4.1_

**Comment by bhack** [comment]
See https://github.com/tensorflow/tensorflow/pull/35189 /cc  @njeffrie @sicong-li-arm

**Comment by Mjonir** [comment]
I'm having this problem as well on Tensorflow 2.3.1. I think it might be due to the network working on 1D data, such as Conv1D layers, which are converted behind the scenes to 2D operations.

**Comment by Xeratec** [comment]
Having the same problem with TensorFlow 2.5.0. I want to use Conv1D layers with dilatation. Is it possible to add this operation to tflite/micro?

**Comment by jensmeder** [comment]
@Xeratec @Mjonir @DominikStachura I have stumbled upon the same issue with TF 2.4.1 but have found a solution/workaround that might help you as well. I have replaced all 1D operations such as Conv1D or MaxPooling1D with their equivalent 2D counterpart. The process is straightforward:

- Replace the 1D operations with their 2D counterpart
- Adjust the input shape for your network
- Adjust the `kernel_size` for your Conv2D layers, e.g., `kernel_size=(3,1)` instead of `kernel_size=3`
- Adjust the `pool_size` for the pooling layers, e.g., `pool_size=(2,1)` instead of `pool_size=2`
- Retrain the network

It's not ideal but it's a solution for now. Hope that helps!
